docs/conf.py

```diff
@@ -28,37 +28,24 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'recommonmark',
+    'myst_parser',
```

Comment:
What is the advantage, just curious

Reply:
nm, I looked it up. Awesome!

Reply:
Aside from the numerous technical benefits of MyST (like better cross-referencing syntax, better math support, etc.), it has a much more responsive developer community. I've reported issues to recommonmark and they've sat there for years without a response. I reported several issues to MyST and they were all addressed right away. This includes one issue which affects this document, and which will be fixed in the next release of MyST. The issue is that markup in link text is not rendered properly.